lab_7/work/src/main/java/ua/lviv/iot/TextClearing/TextClearer.java

```diff
@@ -0,0 +1,71 @@
+package ua.lviv.iot.TextClearing;
+import java.io.FileWriter;
+import java.text.BreakIterator;
+import java.util.ArrayList;
+import java.util.Locale;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class TextClearer{
+    public String txtFileRider(String path) {
+        try{
+            return(new String(Files.readAllBytes(Paths.get(path))));
+        }catch(IOException e){
+            e.printStackTrace();
+        }
+        return null;
+    }
+
+    public boolean areStringContainsQuestion(String str){
```

Comment:
typical naming for a boolean field/method is "is...", "has..." or "can...", indicating you're working with just ONE object. Since you're working with just one string, the correct method name would be **is**StringContainsQuestion. Not a big deal, but would look better :)